addon/components/datepicker-support.js

```diff
@@ -52,6 +52,14 @@ export default Ember.Mixin.create({
         if (!self.$().val()) {
           self._applyDateValue(null);
         }
+      }).
+      on('focusout', function() {
+        console.log('focusout');
+        self.sendAction('focusOut');
```

Comment:
Wouldn't it make sense to follow the same naming convention and use dasherized names instead of camelCased?

```js
on('focusout', function() {
  self.sendAction('focus-out');
}
```

Also it might make sense to pass an event object, so everything works similar to [`Ember.TextSupport`][1] class.

```js
on('focusout', function(event) {
  self.sendAction('focus-out', self, event);
}
```

What do you think @octronic?

[1]: https://github.com/emberjs/ember.js/blob/912b485f174bf3d94b058de15fc17d64cd82d0b0/packages/ember-views/lib/mixins/text_support.js#L269-L272